cli/src/commands/edit.rs

```diff
@@ -21,10 +21,12 @@ use crate::cli_util::{CommandHelper, RevisionArg};
 use crate::command_error::CommandError;
 use crate::ui::Ui;
 
-/// Edit a commit in the working copy
+/// Loads the contents of the specified revision and sets it as the active changeset.
 ///
-/// Puts the contents of a commit in the working copy for editing. Any changes
-/// you make in the working copy will update (amend) the commit.
+/// Note, it is recommended to instead to use `jj new` and `jj amend`. Creating a new
+/// changeset enables being more intentional wirth respect to the existing revision.
+///
+/// For more information, see https://github.com/martinvonz/jj/blob/main/docs/FAQ.md#how-do-i-resume-working-on-an-existing-change
```

Comment:
This does not exist yet, so move it into the next commit.

Reply:
Will do